src/test/java/de/tum/in/www1/artemis/exam/ExamUtilService.java

```diff
@@ -428,6 +428,7 @@ public Exam addExamWithModellingAndTextAndFileUploadAndQuizAndEmptyGroup(Course
 
     public StudentExam addStudentExam(Exam exam) {
         StudentExam studentExam = ExamFactory.generateStudentExam(exam);
+        studentExam.setWorkingTime(exam.getDuration());
```

Comment:
Note: Had to add this so the working time calculation does not throw a `null` pointer. Do you see any problematic implications?

Reply:
🤔 Are there other (production) cases where the working time could be null?

Reply:
Really no clue, but good point really. Guess this invariant is enforced by exam creation / update, so should not happen.